templates/next/components/foo/Show.tsx

```diff
@@ -2,15 +2,15 @@ import { FunctionComponent, useState } from 'react';
 import Link from 'next/link';
 import { useRouter } from "next/router";
 import { fetch } from "../../utils/dataAccess";
-import ReferenceLinks from '../common/ReferenceLinks';
```

Comment:
Why?

Reply:
ReferenceLinks  not used on Show.tsx I just cleaned the code